lib/omniauth/strategies/openid_connect.rb

```diff
@@ -42,6 +42,7 @@ class OpenIDConnect
       option :send_nonce, true
       option :send_scope_to_token_endpoint, true
       option :client_auth_method
+      option :ux
```

Comment:
I'm good to use the name `ux` for now so that you can try get things working for the demo at 3:30, but as discussed in slack, I'm curious if there could be a better one we could use